### PR TITLE
fix: [CI-10165]: Add additional kaniko args

### PR DESCRIPTION
**Pull request by smjt-h**

### CODE REVIEW

File: kaniko.go [hunk 1]
```diff
@@ -224,7 +260,137 @@ func (p Plugin) Exec() error {
...
+
+	if p.Build.CacheRunLayers {
+		cmdArgs = append(cmdArgs, "--cache-run-layers=true")
+	}
+
+	if p.Build.Cleanup {
+		cmdArgs = append(cmdArgs, "--cleanup=true")
+	}
+
+	_, ok := os.LookupEnv("PLUGIN_COMPRESSED_CACHING")
```
Comment (vistaarjuneja) [inline kaniko.go:276]: why are we looking up the env variable?

Comment (smjt-h) [reply kaniko.go:276]: this is because, we want to set the flag to true or false only if its set,

Comment (vistaarjuneja) [reply kaniko.go:276]: Wouldn't `flag.IsSet` return `false` if the flag does not come from the env? And then we could set the pointer accordingly?

Comment (smjt-h) [reply kaniko.go]: I see, I have changed and added this logic, working as expected with true and false both.

**Review by vistaarjuneja — APPROVED** [review]

**Pull request merged**